### PR TITLE
Update the storage recommendations for metrics

### PR DESCRIPTION
**Pull request by jboxman**
@sichvoge 

I've made some changes based on your feedback. Not sure on the exact wording for file storage; Does the metrics stack completely ignore the existence of a file PVC with RWX configured?

Thanks.

### CODE REVIEW

**Comment by sichvoge** [comment]
No it does only ignore the `accessMode` field as it is always set to `ReadWriteOnce`.

**Comment by jboxman** [comment]
@sichvoge, I've updated the language. Can you take a look? Thanks!

**Comment by jboxman** [comment]
@sichvoge, I may omit (_If you are using file storage, do not configure the RWX access mode on any PersistentVolumeClaims configured for use with metrics._) so I can merge this. I don't understand storage well enough to formulate a proper sentence. It's been a few months and I'd like to get this merged into 4.x and 3.x docs.

**Comment by jboxman** [comment]
@openshift/team-documentation, PTAL. Thanks!

File: modules/recommended-configurable-storage-technology.adoc [hunk 1]
```diff
@@ -40,7 +39,7 @@ use shared storage and a single volume (RWX).]
 | Yes
 | Configurable
 | Configurable
-| Configurable footnoteref:[metrics-warning]
+| Configurable footnoteref:[metrics-warning,For metrics, using file storage with the ReadWriteMany (RWX) access mode is unreliable. If you are using file storage, do not configure the RWX access mode on any PersistentVolumeClaims configured for use with metrics.]
```
Comment (kalexand-rh) [inline modules/recommended-configurable-storage-technology.adoc]: s/are using/use
s/configured/that are configured

**Review by kalexand-rh — COMMENTED** [review]
Couple of picks and it LGTM

**Comment by jboxman** [comment]
@kalexand-rh thanks!

**Pull request merged**